pkg/interface/chat/src/js/components/lib/chat-input.js

```diff
@@ -275,14 +277,6 @@ export class ChatInput extends Component {
 
   messageChange(editor, data, value) {
 
-    const mode = editor.getOption('mode');
-    const specialFormat = value.startsWith('#') || value.startsWith('@')
```

Comment:
Can't we support both the button and the `#` prefix? I get that we're not going hard on keyboard shortcuts yet, but right now I can send some code in between talking about the code without taking my hands off the keyboard. We probably shouldn't degrade that into "mouse only".

Reply:
Absolutely, will put it back in. I'd like to visit keyboard shortcuts more broadly in the near future though.